src/NuGet.Core/NuGet.CommandLine.XPlat/NuGet.CommandLine.XPlat.csproj

```diff
@@ -18,7 +18,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Extensions.CommandLineUtils" />
+    <PackageReference Include="Microsoft.Extensions.CommandLineUtils.Sources" PrivateAssets="All" />
```

Comment:
This will cause the package to not be listed as a dependency of this project's package when it's packed. Later when we insert into the .NET SDK, isn't this going to cause a runtime error since this dependency won't be installed?

Reply:
There's actually no assemblies in this package - it's just .cs files that are injected into and compiled as part of the project build.  I cribbed it from [ASP.NET](https://github.com/aspnet/LibraryManager/blob/133eb2052a84f5be6ee3ae9f5ab7e093c6d37174/src/libman/libman.csproj#L17).